resources/core/charts/logging/templates/tests/logging.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: test-{{ template "fullname" . }}
+  labels:
+    helm-chart-test: "true"
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["create", "delete", "get", "list"]
+- apiGroups: [""]
+  resources: ["services", "services/proxy", "configmaps", "pods/log"]
```

Comment:
I guess we dont need access to `"services", "services/proxy", "configmaps",

Reply:
Right, fixed it